### ISSUE TITLE
resync command

### ISSUE THREAD

**Issue by devhawk** [neo-project/neo-express#233]

**Comment by cschuchardt88** [comment]
This would require some parts of `blockchain-toolkit-library` for example `Neo.BlockchainToolkit.Constants` to add a default batch file and policy file

**Comment by Liaojinghui** [comment]
@shargon  can we move https://github.com/N3developertoolkit/neo-blockchaintoolkit-library into the neo-project?

**Comment by Liaojinghui** [comment]
> I prefer it to be in `neo-express` repo

 That would be even better

**Comment by Ashuaidehao** [comment]
I'm maintaining https://github.com/N3developertoolkit/neo-blockchaintoolkit-library.

**Comment by Liaojinghui** [comment]
> I'm maintaining https://github.com/N3developertoolkit/neo-blockchaintoolkit-library.

Then how do you think about the idea of merging toolkit into express?

**Comment by cschuchardt88** [comment]
@Ashuaidehao you know i will help you. But I hate waiting for changes to be released to nuget to get updated package.

**Comment by Ashuaidehao** [comment]
> > I'm maintaining https://github.com/N3developertoolkit/neo-blockchaintoolkit-library.
> 
> @Ashuaidehao Then how do you think about the idea of merging toolkit into express?

I agree it. The only thing is how to make neo-express release github action going on.

**Comment by Ashuaidehao** [comment]
> @Ashuaidehao you know i will help you. But I hate waiting for changes to be released to nuget to get updated package.

Yes, I remember you have do a lot of valuable work in https://github.com/neo-project/neo-express/pull/288 , thanks very much!

**Comment by Liaojinghui** [comment]
> > > I'm maintaining https://github.com/N3developertoolkit/neo-blockchaintoolkit-library.
> > 
> > 
> > @Ashuaidehao Then how do you think about the idea of merging toolkit into express?
> 
> I agree it. The only thing is how to make neo-express release github action going on.

I can run it.

**Comment by Ashuaidehao** [comment]
> > > > I'm maintaining https://github.com/N3developertoolkit/neo-blockchaintoolkit-library.
> > > 
> > > 
> > > @Ashuaidehao Then how do you think about the idea of merging toolkit into express?
> > 
> > 
> > I agree it. The only thing is how to make neo-express release github action going on.
> 
> I can run it.

Great! Do you check if the "[NUGET_TOKEN](https://github.com/neo-project/neo-express/blob/33e2c1db1aa28ccb0d5821bb320952a42deb79b1/.github/workflows/release.yml#L58)" is expired?

**Comment by Liaojinghui** [comment]
Can some one fix this for me:

Run dotnet pack src/worknet --output ./out2 --configuration Release --no-restore --verbosity normal
  dotnet pack src/worknet --output ./out[2](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:2) --configuration Release --no-restore --verbosity normal
  shell: /usr/bin/bash -e {0}
  env:
    DOTNET_ROOT: /home/runner/.dotnet
    GitAssemblyInformationalVersion: [3](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:3).6.17+be7bf33b3b
    GitBuildVersion: 3.6.17.[4](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:4)8763
    GitBuildVersionSimple: 3.6.17
MSBuild version 17.7.4+3ebbd7c49 for .NET
Build started 11/1[5](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:5)/2023 10:39:52.
     1>Project "/home/runner/work/neo-express/neo-express/src/worknet/neoworknet.csproj" on node 1 (pack target(s)).
     1>PrepareForBuild:
         Creating directory "bin/Release/net7.0/".
         Creating directory "obj/Release/net7.0/".
         Creating directory "/home/runner/work/neo-express/neo-express/src/worknet/obj/Release/net7.0/refint/".
         Creating directory "obj/Release/net7.0/ref".
     1>/home/runner/.dotnet/sdk/7.0.404/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(2[6](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:6)6,5): error NETSDK1004: Assets file '/home/runner/work/neo-express/neo-express/src/worknet/obj/project.assets.json' not found. Run a NuGet package restore to generate this file. [/home/runner/work/neo-express/neo-express/src/worknet/neoworknet.csproj]
     1>Done Building Project "/home/runner/work/neo-express/neo-express/src/worknet/neoworknet.csproj" (pack target(s)) -- FAILED.

Build FAILED.

       "/home/runner/work/neo-express/neo-express/src/worknet/neoworknet.csproj" (pack target) (1) ->
       (ResolvePackageAssets target) -> 
         /home/runner/.dotnet/sdk/[7](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:7).0.404/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(266,5): error NETSDK[10](https://github.com/neo-project/neo-express/actions/runs/6876167529/job/18701195718#step:8:11)04: Assets file '/home/runner/work/neo-express/neo-express/src/worknet/obj/project.assets.json' not found. Run a NuGet package restore to generate this file. [/home/runner/work/neo-express/neo-express/src/worknet/neoworknet.csproj]

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.56
Error: Process completed with exit code 1.



Make it like:


- name: Install dependencies
  run: dotnet restore src/worknet

- name: Pack NeoWorkNet 
  run: dotnet pack src/worknet --output ./out2 --configuration Release --no-restore --verbosity normal

**Comment by Liaojinghui** [comment]
@Ashuaidehao @cschuchardt88  please take a look at the new release

**Comment by cschuchardt88** [comment]
@Liaojinghui I think we should add in #336 #335

**Comment by Liaojinghui** [comment]
Good to merge already?

**Comment by cschuchardt88** [comment]
yes

**Comment by cschuchardt88** [comment]
If arent good, than they will be Draft

**Comment by Liaojinghui** [comment]
> If arent good, than they will be Draft

There are conflicts
